content/content-types/src/java/org/sakaiproject/content/types/FolderType.java

```diff
@@ -225,53 +225,6 @@ public boolean allowAddAction(ResourceToolAction action, ContentEntity entity) {
         return true;
     }
 
-    public class FolderPermissionsAction extends BaseInteractionAction {
-
-        public FolderPermissionsAction(String id, ActionType actionType,
-                                       String typeId, String helperId, Localizer localizer) {
-            super(id, actionType, typeId, helperId, localizer);
-        }
-
-        public String initializeAction(Reference reference) {
-            ToolSession toolSession = sessionManager.getCurrentToolSession();
-
-            toolSession.setAttribute(PermissionsHelper.TARGET_REF, reference.getReference());
-
-            // use the folder's context (as a site and as a resource) for roles
-            Collection<String> rolesRefs = new ArrayList<>();
-            rolesRefs.add(siteService.siteReference(reference.getContext()));
-            rolesRefs.add(reference.getReference());
-            toolSession.setAttribute(PermissionsHelper.ROLES_REF, rolesRefs);
-
-            // ... with this description
-            String title = reference.getProperties().getProperty(ResourceProperties.PROP_DISPLAY_NAME);
-            String[] args = {title};
-
-            toolSession.setAttribute(PermissionsHelper.DESCRIPTION, formattedLocalizer("title.permissions", args).getI18nString());
-
-            // ... showing only locks that are prpefixed with this
-            toolSession.setAttribute(PermissionsHelper.PREFIX, "content.");
-
-            return BaseInteractionAction.getInitializationId(reference.getReference(), this.getTypeId(), this.getId());
-        }
-
-        public boolean available(ContentEntity entity) {
-            boolean ok = true;
-            if (entity == null || ContentHostingService.ROOT_COLLECTIONS.contains(entity.getId())) {
-                ok = false;
-            } else if (entity.getId().startsWith(ContentHostingService.COLLECTION_DROPBOX)) {
-                ok = false;
```

Comment:
@adrianfish @ern maybe removing this code is what began to allow the use of "Edit Folder Permissions" in Dropbox tool?